### ISSUE TITLE
ADD ENABLE/DISABLE OPTION TO RMB TO AUTO-AIM!!!

### ISSUE THREAD

**Issue by TheSatanicAngeI** [zbot473/SurvivHacks#164]
pls

**Comment by TheSatanicAngeI** [comment]
Also what the hell happened with plane delay ?

Plane shadow doesn't show on map and delay as been changed.

**Comment by zbot473** [comment]
Ok

**Comment by zbot473** [comment]
Is there any variable with planes?

**Comment by alexvornoffice** [comment]
No, maybe inside objects array (objectCreator), who knows... Or they just renamed planes key to something else...

**Comment by alexvornoffice** [comment]
@TheSatanicAngeI About RMB aim switch, give a try to my new update with collision detection maybe you will like. Shift key to toogle auto aim and to remove already shift key functionality - that will make sense for me.

**Comment by zbot473** [comment]
> No, maybe inside objects array (objectCreator), who knows... Or they just renamed planes key to something else...

Same, i can't find anything

**Comment by TheSatanicAngeI** [comment]
> @TheSatanicAngeI About RMB aim switch, give a try to my new update with collision detection maybe you will like. Shift key to toogle auto aim and to remove current shift key buggy functionality - that will make sense for me.

Shift to not aim is good but not if you have to keep it down lol

Make it a toggle variable or something.

I wish I could help you guys with scripting its a lot to keep this updated :disappointed:
